src/richie/apps/core/management/commands/create_demo_site.py

```diff
@@ -102,6 +102,86 @@
         },
     },
 }
+CATEGORIES_INFO = [
+    {
+        "title": {"en": "Level", "fr": "Niveau"},
+        "children": [
+            {"title": {"en": "Beginner", "fr": "Débutant"}},
+            {"title": {"en": "Advanced", "fr": "Avancé"}},
+            {"title": {"en": "Expert", "fr": "Expert"}},
+        ],
+    },
+    {
+        "title": {"en": "Subject", "fr": "Subjet"},
+        "children": [
```

Comment:
I think it might indeed make more sense to have a more general and uncontroversial taxonomy for the demo site.